BoldGrid/boldgrid-backup
# Emergency Restoration/Standalone CLI Process

This addition is to allow CLI users to perform a check or restoration from either the latest backup or a specified file path.  For now, all that is really requires is the ability to perform a manual restoration.  More functionality (detection, triggers, etc.) will be added later.

To test:
1) In BoldGrid Backup (wp-admin), perform a backup.
2) On the CLI, run: `php cron/bgbkup-cli.php help`
You will see the usage displayed.
`Usage: php bgbkup-cli.php <check|restore> [method=ajax|cli|pclzip|ziparchive] [zip=<path/to/backup.zip>]`
3) On the CLI, run: `php cron/bgbkup-cli.php check`
Everything should be ok, so it should say "Info: No action taken."
4) On the CLI, run: `php cron/bgbkup-cli.php restore`
Your site will be restored using the latest backup created.
5) Create another backup.
6) On the CLI, run: `php cron/bgbkup-cli.php restore zip=<path/to/zip>`
You may want to specify the path to the first backup that you created; not the latest.
Your site will be restored using the specified backup ZIP file.
The dialog will reference the file path used.
